public/css/auth.css

```diff
@@ -0,0 +1,97 @@
+/* BACKGROUND IMAGE */
+
+body {
+    background: url("/public/images/background_aufschlag.jpg") no-repeat center
+        center fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    background-size: cover;
+    -o-background-size: cover;
+}
+
+.overlay {
+    position: absolute;
+    min-height: 100%;
+    min-width: 100%;
+    left: 0;
+    top: 0;
+    background: rgba(0, 0, 0, 0.6);
+}
+
+/* HEADER */
+
+.navbar {
+    background-color: #fdfb50;
+    overflow: hidden;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    justify-content: space-around;
+}
+
+.image {
+    max-width: 40%;
+}
+
+.navbar a {
+    float: left;
+    display: block;
+    color: #0275d8;
+    text-align: center;
+    text-decoration: none;
+    font-size: 17px;
+}
+
+h3 {
+    color: #fdfb50;
+    text-align: center;
+}
+
+/* LOGIN-FORM / REGISTER-CARD */
+
+label {
+    color: #f5f5f5;
+}
+
+/* input {
+    background: #fdfb50 !important;
+  } */
+
+.login-form {
+    left: 50%;
+    top: 51%;
+
+    margin: auto;
+    padding: 0.5rem;
+
+    position: absolute;
+
+    text-align: center;
+
+    transform: translate(-50%, -49%);
+
+    width: 100%;
+    max-width: 20rem;
+}
+
+/* LOGIN-/REGISTER-BUTTON */
+
+.btn {
+    border: 1px solid #fdfb50;
+    color: #fdfb50;
+}
+
+.btn:hover {
+    border: 1px solid #f5f5f5;
+    background: #f5f5f5;
+    color: #0275d8;
+}
+
+/* colors */
+
+colors {
+    color: #000; /* black */
```

Comment:
You can turn these comments into variables. Put them in your main.css and use them all over the place. Makes it much easier to keep colors consistent